Seminars/README.md

```diff
@@ -0,0 +1,27 @@
+Welcome to our TRIC-DT Seminar Series!
+**Take part** in this community by attending the [next event](#next-events), [propose](#propose-a-topic) or upvoting a new topic or [sign up to present](#sign-up-as-a-presenter)!
+
+# 'Crafting Digital Twins' Seminar Series
+
+The “Crafting Digital Twins: Computational Methods Across Environment, Health & Infrastructure” seminar series will be an interdisciplinary platform for TRIC-DT researchers to share and discuss about the computational methods, algorithms, and models that underpin Digital Twin technology in diverse fields. This series will provide a space for exchange of knowledge and methodologies across the themes of environment, health, and infrastructure, with the aim to cultivate a unified vocabulary and uncover synergies in our work across TRIC-DT.
+
+By uniting researchers to discuss the methodological hurdles in their foundational work, this series will offer immediate value to each team member, enabling you to acquire practical insights applicable to projects at any stage, no matter how “DT-ready”. Beyond the technical learnings, presenting to a broader peer group allows us to see common challenges from fresh perspectives, potentially igniting innovative solutions to our problems.
```

Comment:
I wonder if we want to address "team members" here - this isn't a series only for the TRIC team, right? If it is, disregard my comment. Otherwise: I think team members here might make people a little nervous about whether they "are allowed" to participate or not.

Reply:
I do really love the low technological bar set here, though! It really feels inviting and fresh to me. Also energetic in the "igniting innovative solutions" - great!